code/src/gui/gui_glview.cpp

```diff
@@ -0,0 +1,201 @@
+/**************************************************************************************************/
+/**
+* \addtogroup <DoxModuleId>
+* @{
+* \details
+* This file provides the public interface for the <Module_Name> Module.
```

Comment:
Change this comment.

Reply:
Fixed https://github.com/tdiego1/worldweaver/pull/19/commits/55657244dc8a411562b9c4ebfc0d9806e8797d13